source/includes/steps-fle-convert-to-a-remote-master-key-azure.yaml

```diff
@@ -126,6 +126,31 @@ content: |
                   "clientSecret": "<Azure client secret>"
               }
            }
+
+     .. tab::
+        :tabid: csharp
+
+        .. code-block:: chsarp
+
+           var kmsProviders = new Dictionary<string, IReadOnlyDictionary<string, object>>();
+
+           var azureTenantId = Environment.GetEnvironmentVariable("FLE_AZURE_TENANT_ID");
+           var azureClientId = Environment.GetEnvironmentVariable("FLE_AZURE_CLIENT_ID");
+           var azureClientSecret = Environment.GetEnvironmentVariable("FLE_AZURE_CLIENT_SECRET");
+           var azureKmsOptions = new Dictionary<string, object>
+           {
+               { "tenantId", azureTenantId },
```

Comment:
As I understand, it's better to mention all available options(with possible description in comments). See here: https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/client-side-encryption.rst#kmsproviders. I also included it in the patch